src/components/features/product-table.tsx

```diff
@@ -19,7 +19,7 @@ import {
 import { Typography } from '@/components/atoms/typography.tsx'
 import { PaginationSpec } from '@/schemas/common.ts'
 import { ProductVendor } from '@/schemas/product.ts'
-import { noop, formatPrice } from '@/utils/common.ts'
+import { formatPrice, noop } from '@/utils/common.ts'
 
 import { Button } from '../atoms/button.tsx'
 import CustomPagination from '../molecules/custom-pagination.tsx'
```

Comment:
I know this isn't your change, but could you please change it to use '@' instead of relative imports?